sdk/src/Core/Amazon.Runtime/Pipeline/_unity/UnityWebRequestWrapper.cs

```diff
@@ -68,6 +68,7 @@ static UnityWebRequestWrapper()
 
             setRequestHeaderMethod = unityWebRequestType.GetMethod("SetRequestHeader");
             sendMethod = unityWebRequestType.GetMethod("Send");
+            sendMethod = sendMethod ?? unityWebRequestType.GetMethod("SendWebRequest"); // from Unity 2017.2
```

Comment:
Can we first try to get the method SendWebRequest and then Send as shown below? If the method Send is introduced to the class again in the future but with different functionality, this will make sure that we will not use the wrong method here.
       
           sendMethod = unityWebRequestType.GetMethod("SendWebRequest"); // from Unity 2017.2
           sendMethod = sendMethod ?? unityWebRequestType.GetMethod("Send");

Reply:
OK, that's right.